### PR TITLE
Mobile version of Portfolio wroks 

### PR DESCRIPTION
**Pull request by wajahatalli**
## Portfolio
In this Pull Request, I:
- Created branches with the **setup-files-mobile** branch.
- Followed GitHub flow.
- Added a descriptive and easy-to-understand **README.md** file
- Added a HTML file
- Added a title of "Portfolio"
- Added Logo and Hamburger-menu
- Added headline & headline text, including a short description
- Added some social buttons
- Added a CSS File
- Added required styles in the CSS file 
- Made required changes in HTML file
- Added social media account links to social icons in HTML
- Made some minor changes in CSS file
- Improved README.md file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                        <li class="social-icons-li">
+                            <a href="#"><i class="fa-brands fa-facebook"></i></a>
+                        </li>
+                        <li class="social-icons-li">
+                            <a href="#"><i class="fa-brands fa-medium"></i></a>
+                        </li>
+                        <li class="social-icons-li">
+                            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                        </li>
+                    </ul>
```
Comment (Felix45) [inline index.html:53]: [Required] Please add links to all the social media accounts listed here, most importantly you should include the following links `Github profile`, `LinkedIn` & `Twitter` having active links to your social media accounts makes it easy for recruiters to reach out to you.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins&display=swap" rel="stylesheet"/>
+    <script src="https://kit.fontawesome.com/79b3da0f3c.js" crossorigin="anonymous"></script>
+    <title>Portfolio</title>
+</head>
+<body>
+    <div class="wrap">
+        <header>
+            <nav class="navigation">
+                <a href="#">
+                    <p class="text-logo">John Doe</p>
```
Comment (Felix45) [inline index.html:19]: [ Required ] Please remove the `p` tag around the logo, semantically this is not correct. Wrapping the logo section in an `a`  tag is the best practice. please rectify this.

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @wajahatalli , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :heavy_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 
- [ ] Descriptive commit messages :white_check_mark: 
- [ ] Github flow is correctly used :white_check_mark: 
- [ ] Application works as expected. :white_check_mark: 

almost-there
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Yerimah — APPROVED** [review]
## Status: Approved 👍 

Hi @wajahatalli, 👋  

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f474b3767725a594c47376373302f67697068792e676966

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed correctly ✅ 
- Pull Request is descriptive ✅ 
- Project is working fine ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**